<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="36" started-at="2016-09-27T17:42:31Z" finished-at="2016-09-27T17:42:31Z">
    <groups>
      <group name="Smoke">
        <method signature="HR_SmokeTest_Emp.RehireMultipleEmp()[pri:18, instance:com.HRM.testcases.HR_TC.Employees.HR_SmokeTest_Emp@68bbe345]" name="RehireMultipleEmp" class="com.HRM.testcases.HR_TC.Employees.HR_SmokeTest_Emp"/>
      </group> <!-- Smoke -->
    </groups>
    <test name="Default test" duration-ms="36" started-at="2016-09-27T17:42:31Z" finished-at="2016-09-27T17:42:31Z">
      <class name="com.HRM.testcases.HR_TC.Employees.HR_SmokeTest_Emp">
        <test-method status="PASS" signature="StartBrowser_NavURL()[pri:0, instance:com.HRM.testcases.HR_TC.Employees.HR_SmokeTest_Emp@68bbe345]" name="StartBrowser_NavURL" is-config="true" duration-ms="10" started-at="2016-09-27T23:12:31Z" finished-at="2016-09-27T23:12:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- StartBrowser_NavURL -->
        <test-method status="FAIL" signature="RehireMultipleEmp()[pri:18, instance:com.HRM.testcases.HR_TC.Employees.HR_SmokeTest_Emp@68bbe345]" name="RehireMultipleEmp" duration-ms="5" started-at="2016-09-27T23:12:31Z" finished-at="2016-09-27T23:12:31Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.HRM.testcases.HR_TC.Employees.HR_SmokeTest_Emp.RehireMultipleEmp(HR_SmokeTest_Emp.java:2929)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- RehireMultipleEmp -->
        <test-method status="FAIL" signature="ClosingBrowser()[pri:0, instance:com.HRM.testcases.HR_TC.Employees.HR_SmokeTest_Emp@68bbe345]" name="ClosingBrowser" is-config="true" duration-ms="0" started-at="2016-09-27T23:12:31Z" finished-at="2016-09-27T23:12:31Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.HRM.Utils.CommonFunctions.closeBrowser(CommonFunctions.java:784)
	at com.HRM.testcases.HR_TC.Employees.HR_SmokeTest_Emp.ClosingBrowser(HR_SmokeTest_Emp.java:74)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:216)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:143)
	at org.testng.TestRunner.afterRun(TestRunner.java:973)
	at org.testng.TestRunner.run(TestRunner.java:604)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ClosingBrowser -->
      </class> <!-- com.HRM.testcases.HR_TC.Employees.HR_SmokeTest_Emp -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
